services/ui-src/src/components/forms/AdminDashSelector.tsx

```diff
@@ -37,7 +37,7 @@ export const AdminDashSelector = () => {
     },
     {
       value: "TA",
-      label: "Timely Access Report (TA)",
+      label: "Timely Access Compliance Measure Report (TACM)",
```

Comment:
Maybe it should look like this? Not sure if it just depends on the context or where it's placed